core/trino-main/src/main/java/io/trino/operator/scalar/ArrayIntersectFunction.java

```diff
@@ -78,23 +78,23 @@ public Block intersect(
             pageBuilder.reset();
         }
 
-        TypedSet rightTypedSet = createDistinctTypedSet(type, elementIsDistinctFrom, elementHashCode, rightPositionCount, "array_intersect");
+        TypedSet rightValues = createDistinctTypedSet(type, elementIsDistinctFrom, elementHashCode, rightPositionCount, "array_intersect");
         for (int i = 0; i < rightPositionCount; i++) {
-            rightTypedSet.add(rightArray, i);
+            rightValues.add(rightArray, i);
         }
 
-        BlockBuilder blockBuilder = pageBuilder.getBlockBuilder(0);
+        BlockBuilder output = pageBuilder.getBlockBuilder(0);
```

Comment:
the original naming is a common pattern. i don't want to bike shed on this, but it feels we're changing something that's subjective into a different, subjective form.

is this at least unifying Array functions implementations between each other? let's add rationale to the commit message, or drop the commit

Reply:
Out of all of these commits, this was the one I was the least sure about, and I almost didn't include it at all; I'll just drop it.